### ISSUE TITLE
why not test on composition-1k

### ISSUE THREAD

**Issue by xxcoco763** [vivoCameraResearch/SDMatte#7]
Thanks for the good work, while i found that you train on the composition-1k, and notes that it does not support composition test. I wonder why?

**Comment by LongfeiH** [comment]
The test set of composition-1k is synthetic and exhibits a considerable gap from real-world data, making it difficult to reliably evaluate model performance. In fact, recent trimap-free approaches rarely adopt composition-1k as a testing benchmark. Nevertheless, due to the limited availability of training datasets in image matting, composition-1k is still inevitably employed as a training set.
